bin/check-http-json.rb

```diff
@@ -73,6 +73,7 @@ class CheckJson < Sensu::Plugin::Check::CLI
   option :whole_response, short: '-w', long: '--whole-response', boolean: true, default: false
   option :dump_json, short: '-d', long: '--dump-json', boolean: true, default: false
   option :pretty, long: '--pretty', boolean: true, default: false
+  option :response_code, long: '--response-code CODE', default: '^2'
```

Comment:
please include a description ( I know not all the others do in this check but in general we try to make sure there is a description. In this case I would suggest including an indication that this accepts regex as opposed to a string. Also I am wondering after seeing #98 if we should while we are at it allow specifying multiple acceptable responses. That being said you could accomplish that with a slightly more complex regex such as `'^(200|429)'` while the other method of specifying multiple via comma separated is less powerful its also easier to use. I am open to both approaches so feel free to talk me into either approach, the one thing I would like to do is match it both locations so people are less likely to have issues if they use both.

Reply:
The regex is more powerful and already used by other options like --query so the extra complexity shouldn't be a burden.

I will add it to check-http.rb and update the PR.